modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -0,0 +1,80 @@
+= Configuring Multiple Credential Providers
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: credential providers, external client providers
+
+With Anypoint Platform, you can use multiple client providers, also known as identity providers (IdP) to manage client credentials to help you implement security policies and regulations. This ensures that client applications that are created to access APIs are stored in different systems based on the environment where API is located. Separate client providers helps with segregating client application management for:
+
+* Different business groups
+* Different environments, such as production, QA, and sandbox
+* Internal and external APIs
+
+For example, a “sandbox” client, whose credentials live in an external Okta IdP, can have contracts only with API instances in the organization “Sandbox A” environment of the business group A and “open” environment of business group B. Similarly, a “core-internal” client, whose credentials live in the private ADFS IdP of the organization, can have contracts only with API instances in the “prod” environment of the business group X. 
+
+
+== How This Works
```

Comment:
@priyaayyar @tbruno this section is really weak and lacks context. It needs more information on how configuration is shared between APIM and Access Management.

Reply:
Need help on this one.